src/main/util.ts

```diff
@@ -19,7 +19,7 @@ export const didPropsChange = <P extends { [key: string]: any}>(prevProps: P, ne
     })
 }
 
-export const isSeat = (obj: SelectableObject) => obj.objectType === 'Seat'
+export const isSeat = (obj: SelectableObject): obj is Seat => obj.objectType === 'Seat'
```

Comment:
I just noticed that the `Seat` import in this file was never used, and I think you guys may have forgotten to add the actual type guard into this function after you changed the type names.

I don't think these type guards are really necessary anymore for the most part after your team took my recommendation on switching to discriminated unions (at least I haven't had any need for them), but it makes more sense for them to at least be accurate if they are here!

Reply:
Thanks, we had indeed missed that! The type guards can still be useful if you need to access certain properties only available on the specific object types.

Reply:
True, I just meant that now that they're all discriminated unions, you can just narrow the type by doing this instead:

```ts
if (obj.objectType === "Seat") {
  console.log(obj.seatOnlyAttribute);
}
```

which is much safer than a type guard generally anyway.

for example:

<img width="316" alt="image" src="https://github.com/seatsio/seatsio-react/assets/9214195/23b870e6-6f76-41d7-adfb-1c223307e3b4">

<img width="297" alt="image" src="https://github.com/seatsio/seatsio-react/assets/9214195/d2edae3a-7992-4174-b4c5-df52ef3a8cf8">

Type guards are really a way to cast a type without manually casting `as Seat`.